app/assets/stylesheets/_vocab-sidebar.scss

```diff
@@ -56,6 +56,7 @@
   &__image {
     min-height: 5rem;
     max-width: 85%;
+    height: 5.625rem;
```

Comment:
Can we make this a rounder number like 5.5 or 6? 5.625 seems a little finicky unless theres a reason that it needs to be that size.

Reply:
Ok sure I will make it 5.5rem. The only reason was that 5.625 matched 90px exactly